src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/plugin.json

```diff
@@ -1,12 +1,13 @@
 {
-    "ID":"F737A9223560B3C6833B5FFB8CDF78E5",
-    "ActionKeywords": [ "*", "<"],
-    "Name":"Window Walker",
-    "Description":"Alt-Tab alternative enabling searching through your windows.",
-    "Author":"betadele",
-    "Version":"1.0.0",
-    "Language":"csharp",
-    "Website":"https://www.windowwalker.com/",
-    "ExecuteFileName":"Microsoft.Plugin.WindowWalker.dll",
-    "IcoPath":"Images\\windowwalker.dark.png"
+  "ID": "F737A9223560B3C6833B5FFB8CDF78E5",
+  "ActionKeyword": "<",
+  "IsGlobal": true,
+  "Name": "Window Walker",
+  "Description": "Alt-Tab alternative enabling searching through your windows.",
+  "Author": "betadele",
+  "Version": "1.0.0",
+  "Language": "csharp",
+  "Website": "https://www.windowwalker.com/",
+  "ExecuteFileName": "Microsoft.Plugin.WindowWalker.dll",
+  "IcoPath": "Images\\windowwalker.dark.png"
```

Comment:
Should we add a second value to differentiate between dark mode and light mode icon?
The icon will be used later in the plugin list in the PT settings.